SensorState.py

```diff
@@ -23,6 +23,7 @@ def __init__(self):
         self.terabee_bot: List[int] = [1]*20
         self.terabee_mid: List[int] = [1]*20
         self.terabee_top: List[int] = [1]*20
+        self.sensor_data = [self.terabee_bot, self.terabee_mid, self.terabee_top, self.lidar]
```

Comment:
any particular reason this is necessary? since the attributes are already accessible